hello-world/tacocat01/.gitignore

```
@@ -0,0 +1 @@
+*.swp
```

Don't forget to include binary files and Cmake artifacts to your gitignore!
```suggestion
*.swp
CMakeCache.txt
CMakeFiles/
Makefile
cmake_install.cmake
helloworld
```